core/modules/qdisp/QdispPool.cc

```diff
@@ -228,6 +228,8 @@ QdispPool::QdispPool(int poolSize, int largestPriority, std::vector<int> const&
         LOGS(_log, LOG_LVL_INFO, "creating priQ pri=" << pri << " min=" << minRun << " max=" << maxRun);
         _prQueue->addPriQueue(pri, minRun, maxRun);
     }
+    // This pool does not kick threads out when they take time (but little CPU) to process,
+    // so maxPoolThreads is just slightly larger than poolSize.
```

Comment:
What's a definition of "slightly"? Could it be expressed in a fraction of the {poolSize}}, or in a fixed number? Is there any chance thread allocation may "go wild" and exceed all threads available to a process (which is a little less than 30k)?